### ISSUE TITLE
RP freezes when using tab, etc. to select different kernel

### ISSUE THREAD

**Issue by MarioG-X** [dakanji/RefindPlus#85]
### RefindPlus Version

v0.13.2.AJ Release

### Device Type

Apple Mac

### Problem Description

I did not use tab/insert in AJ until today, it turns out RP simply freezes on a 2014 Mac mini and 2012 iMac _immediately by pressing tab_ to get options. You must power of/on to get back. I am just reporting so you know to fix it in next release unless already done.

### Problem Point

RefindPlus (Before starting loader/tool)

### Affected Items

Lin OS Loader

### Debug Log and (Optional) Screenshots

Not able to get log since it requires debug version

### Additional Context

Not urgent since I was able to go back to refind original an it worked so I could get linux back to working. Just reporting so you know it's an issue with AJ. You can view my first reported problem if you need a detailed description of machine configuration but since it occurs on two different types of Macs I don't think machine config is too important.

**Comment by dakanji** [comment]
Thanks for the report

> Not able to get log since it requires debug version

Yes, the release version does not generate a log and the idea is to use the debug version to generate such.

**Comment by MarioG-X** [comment]
Dumb me, I forgot DBG was part of the AJ folder. Here is the log. BTW, I noticed at the bottom of log it says arrow left, but I did not press shift so should have been arrow right? I also tried on an iMac with the Apple keyboard so I don't think the actual key makes a difference. It looks like it got they key but froze before displaying options menu.
[21v22t5454.log](https://github.com/dakanji/RefindPlus/files/7583363/21v22t5454.log)

**Comment by dakanji** [comment]
Thanks. What happens when you use the Tab key on someone OS Loader? That is, does this only affect Linux or do others also freeze?

**Comment by MarioG-X** [comment]
Results: oddly it works for all the macOS systems displayed. It freezes when I press Tab for the 2 Opensuse and 1 Elementary partitions, all 3 Linux.

**Comment by dakanji** [comment]
I see. Needs looking into further then.

Please run the attached and share the log. Thanks

[X178-BOOTx64.zip](https://github.com/dakanji/RefindPlus/files/7583819/X178-BOOTx64.zip)

**Comment by MarioG-X** [comment]
That one Tab works. I was able to see options. Log:
[21v22y2006.log](https://github.com/dakanji/RefindPlus/files/7584736/21v22y2006.log)

**Comment by dakanji** [comment]
The issue is linked to a memory management bug that was fixed after the last release.
You can build the current code base pending the next release.
